release-content/0.14/migration-guides/12234_Move_commands_module_into_bevyecsworld.md

```diff
@@ -1,8 +1,9 @@
-`Command` and `CommandQueue` have been moved from `bevy::ecs::system` to `bevy::ecs::world`.
+`Command` and `CommandQueue` have been moved from `bevy::ecs::system` to `bevy::ecs::world`. If you import them directly, you will need to update your import statements. (This does not affect you if you just import the prelude.)
```

Comment:
Do we need to explicitly mention the case where the user is not affected?

Reply:
Fixed in 450589a1c3adb1d3c2c3870cb22c800c38058160.